### DISCUSSION TITLE
ssh cert key support

### DISCUSSION THREAD

**Discussion by Capri205** [wezterm/wezterm#595] [Q&A] [answered]
Hi. Does wezterm support private/public key support for ssh login? I couldn't find anything in the documentation or command help.

Thanks.

**Comment by wez** [comment, marked as answer]
Support for key files landed in the nightly builds recently: the section at the bottom of [the ssh docs](https://wezfurlong.org/wezterm/ssh.html) explains what is supported.

**Reply by Capri205** [reply]
Thanks a lot for the prompt reply Wez! I will look into it. Love your work!
